test_fixtures/align_content_space_around_single_line.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <script src="../scripts/gentest/test_helper.js"></script>
+  <link rel="stylesheet" type="text/css" href="../scripts/gentest/test_base_style.css">
+  <title>
+    Test description
+  </title>
+<head/>
+<body>
+
+<div id="test-root" style="width: 100px; height: 100px; flex-direction: row; flex-wrap: wrap; align-content: space-around;">
+  <div style="height: 10px; width: 50px"></div>
+  <div style="height: 10px; width: 50px"></div>
+  <div style="height: 10px; width: 50px"></div>
+  <div style="height: 10px; width: 50px"></div>
+  <div style="height: 10px; width: 50px"></div>
+  <div style="height: 10px; width: 50px"></div>
+</div>
+
+</body>
+</html>
```

Comment:
GitHub complains that the test fixtures are missing the trailing newline, is that intentional?

Reply:
I don't think Github is complaining, so much as just pointing it out? The checks seem to be passing fine. In any case, newlines at the end of the document make no actual difference (and none of the other fixtures have them, so it's even consistent!)